extensions/ql-vscode/src/databases/db-config.ts

```diff
@@ -1,10 +1,25 @@
 // Contains models for the data we want to store in the database config
 
 export interface DbConfig {
+  databases: DbConfigDatabases;
+  selected?: SelectedDbItem;
+}
+
+export interface DbConfigDatabases {
   remote: RemoteDbConfig;
   local: LocalDbConfig;
 }
 
+export interface SelectedDbItem {
+  kind: SelectedDbKind;
+  value: string;
+}
+
+export enum SelectedDbKind {
```

Comment:
I think this should be `SelectedDbItemKind` since we've gone with `SelectedDbItem`.

Reply:
👍